controllers/csm_controller.go

```diff
@@ -888,7 +893,7 @@ func (r *ContainerStorageModuleReconciler) PreChecks(ctx context.Context, cr *cs
 			}
 
 		} else {
-			if ischeckingOwnRef { // TODO(Michael): should be deprecated after 0.2.0
+			if ischeckingOwnRef {
```

Comment:
Why are we keeping these lines when the comment says it should be deprecated after 0.2.0

Reply:
that should be another story where we handle the owner reference in a different way 
I will create a story for that